cpp/tensorrt_llm/kernels/contextFusedMultiHeadAttention/fused_multihead_attention_common.h

```diff
@@ -66,9 +66,8 @@ enum class ContextAttentionMaskType
     PADDING = 0,
     // Mask the padded tokens and all the tokens that come after in a sequence.
     CAUSAL,
-    // Causal mask + mask the beginning tokens based on sliding_window_size
-    // Only pay attention to [max(0, q_i - sliding_window_size), q_i]
-    SLIDING_WINDOW_CAUSAL,
+    // Causal mask + attend to the specific sliding window or chunk.
+    SLIDING_OR_CHUNKED_CAUSAL,
```

Comment:
There's a corresponding enum on the PyTorch side that we should update. I'll do it in my followup